neurom/fst/_neuronfunc.py

```diff
@@ -33,7 +33,12 @@
 from neurom.core.types import NeuriteType
 from neurom.core.types import tree_type_checker as is_type
 from neurom.core.dataformat import COLS
-from neurom import morphmath as mm
+from neurom import morphmath
+
+
+def neurons_pop(nrns):
```

Comment:
ahhh. pop was refering to population. I believed that it was pop like list.pop()
ok for neuron_population. I don't have a better name.